### PR TITLE
Made some changes to the conditionals section of the course

### PR DESCRIPTION
**Pull request by susiecoleman**
Ping for a look @gu-lseg/lseg :-)

### CODE REVIEW

File: source/conditionals.rst [hunk 1]
```diff
@@ -17,7 +20,7 @@ Furthermore conditional loops enable us to harness logic relating to repetition.
 Code Blocks
 ===========
 
-A block of code is code that will execute together. A block is defined by the 
+A block of code is code that will execute together. In python a block is defined by the
```
Comment (gtrufitt) [inline source/conditionals.rst]: Minor, but I've been trying to keep it **P**ython for language to differentiate from **p**ython the interpreter.

File: source/conditionals.rst [hunk 2]
```diff
@@ -33,18 +36,22 @@ outcome of the conditional expression that guards their execution.
...
         result = a + 6
 
+
 .. tip::
 
     In other languages code blocks are defined by the use of braces `{}`s
 
+Questions
+-------
+1. Can you change the code above to make the `else` block execute?
```
Comment (gtrufitt) [inline source/conditionals.rst:46]: Nice

File: source/conditionals.rst [hunk 3]
```diff
@@ -61,14 +68,20 @@ value to be equal:
     >>> 5 == '5'
     False
 
+Questions
+-------
+1. Write some equality statements of your own
+
+2. Instead of == you can also use `!=` `<=` `<` `>=` or `>`. Write some equality statements using these
```
Comment (gtrufitt) [inline source/conditionals.rst:75]: I wonder if we should have an 'extra' part about strict comparison in JS - even just a link in the appendix or further learnings somewhere?

File: source/conditionals.rst [hunk 4]
```diff
@@ -103,31 +119,22 @@ visualising execution
     <iframe width="800" height="500" frameborder="0" src="http://pythontutor.com/iframe-embed.html#code=import+random%0Awarm+%3D+20%0Atemperature+%3D+18%0Awhile+temperature+%3C%3D+warm%3A%0A++++print('Its+'+%2B+str(temperature)+%2B+'+degrees.+Wrap+up.')%0A++++temperature+%3D+random.randrange(5,+25)%0A++++%0Aprint('Thats+all+for+now')&origin=opt-frontend.js&cumulative=false&heapPrimitives=false&drawParentPointers=false&textReferences=false&showOnlyOutputs=false&py=3&rawInputLstJSON=%5B%5D&curInstr=0&codeDivWidth=350&codeDivHeight=400"> </iframe>
 
 .. tip::
-    Use while if you don't when you only know a loop will terminate in a given
-    condition.
+    A while loop is used repeatedly execute an instruction until a condition
```
Comment (gtrufitt) [inline source/conditionals.rst]: repeatedly **to**

File: source/conditionals.rst [hunk 5]
```diff
@@ -147,14 +154,29 @@ Europe  UK
 42      8
 ======  =====
 
+Extension: Can you return an error message for the user if they enter an invalid shoe size.
+
+Practical: Loan
```
Comment (gtrufitt) [inline source/conditionals.rst:159]: I wonder if we need a 'tip' section here to help nudge towards writing out the problem on paper (ie: work out the maths before trying to write it in code). That may just be my personal way of programming though...

**Comment by gtrufitt** [comment]
Aces, couple of points, but looks good! 👍

File: source/conditionals.rst (hunk 3 above)
Comment (NataliaLKB) [inline source/conditionals.rst:75]: Lets not make things more confusing by bringing JavaScript into it 😉

File: source/conditionals.rst [hunk 6]
```diff
@@ -92,6 +105,9 @@ example
     cold
     cold
 
+Questions
```
Comment (NataliaLKB) [inline source/conditionals.rst:108]: I love these questions!

**Comment by NataliaLKB** [comment]
Great work! 👍

File: source/conditionals.rst (hunk 1 above)
Comment (susiecoleman) [inline source/conditionals.rst]: fixed

File: source/conditionals.rst (hunk 4 above)
Comment (susiecoleman) [inline source/conditionals.rst]: fixed

File: source/conditionals.rst (hunk 5 above)
Comment (susiecoleman) [inline source/conditionals.rst:159]: fixed

**Pull request merged**